### ISSUE TITLE
[Bug] [Flatcar] Not set Python interpreter on existing VM

### ISSUE THREAD

**Issue by Tomorrow9** [vmware/ansible-vsphere-gos-validation#91]
##### SUMMARY

```

2021-07-06 01:06:40,006 | Failed at Play [ovt_verify_install] ************************

2021-07-06 01:06:40,006 | TASK [Collect filtered guest information for 'xx.xx.xx.xx'] 
task path: /root/ansible-vsphere-gos-validation/common/get_system_info.yml:22
fatal: [localhost -> xx.xx.xx.xx]: FAILED! => {
    "changed": false,
    "module_stderr": "Shared connection to xx.xx.xx.xx closed.\r\n",
    "module_stdout": "/bin/sh: /usr/bin/python: No such file or directory\r\n",
    "msg": "The module failed to execute correctly, you probably need to set the interpreter.\nSee stdout/stderr for the exact error",
    "rc": 127,
    "warnings": [
        "No python interpreters found for host localhost (tried ['/usr/bin/python', 'python3.9', 'python3.8', 'python3.7', 'python3.6', 'python3.5', 'python2.7', 'python2.6', '/usr/libexec/platform-python', '/usr/bin/python3', 'python'])"
    ]
}
```

##### ISSUE TYPE
- Bug Report

##### ANSIBLE VERSION

```paste below



```

##### CONFIGURATION

```paste below



```

##### ENVIRONMENT / OS

Flatcar

##### STEPS TO REPRODUCE

On existing Flatcar VM.

##### EXPECTED RESULTS

Found Python interpreter.

##### ACTUAL RESULTS

Not found Python interpreter.

**Comment by Tomorrow9** [comment]
Filed an issue to Ansible https://github.com/ansible/ansible/issues/75280, to see if it can be fixed when set to discover Python interpreter automatically.

**Comment by Tomorrow9** [comment]
1. New parameter added in Ansible 2.12, we can take a look if this parameter works here.

2. Since "/opt/bin" is not in the PATH, so Ansible not found it's path by ssh commands below:
```
<xx.xx.xx.xx> Attempting python interpreter discovery
<xx.xx.xx.xx> ESTABLISH SSH CONNECTION FOR USER: root
<xx.xx.xx.xx> SSH: EXEC sshpass -d10 ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'User="root"' -o ConnectTimeout=10 -o ControlPath=/root/.ansible/cp/747ef39268 10.191.131.95 '/bin/sh -c '"'"'echo PLATFORM; uname; echo FOUND; command -v '"'"'"'"'"'"'"'"'/usr/bin/python'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.9'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.8'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.7'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.6'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.5'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python2.7'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python2.6'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'/usr/libexec/platform-python'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'/usr/bin/python3'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python'"'"'"'"'"'"'"'"'; echo ENDFOUND && sleep 0'"'"''
**<10.191.131.95> (0, b'PLATFORM\nLinux\nFOUND\nENDFOUND\n', b'')**
```

**Comment by keirazhang** [comment]
By appending /opt/bin into PATH environment variable and put it into /etc/environment, this issue can be resolved.
```
localhost ~ # cat /etc/environment 
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/opt/bin
```

Then run testing with vm_python=auto. and we can see it could find /opt/bin/python.
```
<10.182.136.6> SSH: EXEC sshpass -d12 ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o Port=22 -o 'User="root"' -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ControlPath=/home/qiz/.ansible/cp/ef225d8ec0 10.182.136.6 '/bin/sh -c '"'"'echo PLATFORM; uname; echo FOUND; command -v '"'"'"'"'"'"'"'"'/usr/bin/python'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.9'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.8'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.7'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.6'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.5'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python2.7'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python2.6'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'/usr/libexec/platform-python'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'/usr/bin/python3'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python'"'"'"'"'"'"'"'"'; echo ENDFOUND && sleep 0'"'"''
<10.182.136.6> (0, b'PLATFORM\nLinux\nFOUND\n/opt/bin/python\nENDFOUND\n', b'')
<10.182.136.6> ESTABLISH SSH CONNECTION FOR USER: root
<10.182.136.6> SSH: EXEC sshpass -d12 ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o Port=22 -o 'User="root"' -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ControlPath=/home/qiz/.ansible/cp/ef225d8ec0 10.182.136.6 '/bin/sh -c '"'"'/opt/bin/python && sleep 0'"'"''
<10.182.136.6> (0, b'{"platform_dist_result": ["", "", ""], "osrelease_content": "NAME=\\"Flatcar Container Linux by Kinvolk\\"\\nID=flatcar\\nID_LIKE=coreos\\nVERSION=2765.2.6\\nVERSION_ID=2765.2.6\\nBUILD_ID=2021-06-15-2026\\nPRETTY_NAME=\\"Flatcar Container Linux by Kinvolk 2765.2.6 (Oklo)\\"\\nANSI_COLOR=\\"38;5;75\\"\\nHOME_URL=\\"https://flatcar-linux.org/\\"\\nBUG_REPORT_URL=\\"https://issues.flatcar-linux.org\\"\\nFLATCAR_BOARD=\\"amd64-usr\\"\\n"}\n', b'')
```

**Comment by Tomorrow9** [comment]
Thanks, it works as expected. So we can add this in the README file for one of the exsiting VM prerequtsites, specific to the VM in which Python is not installed in the normal path, and this path is not in the PATH env.

**Comment by keirazhang** [comment]
Issue has been fixed in #108
